src/Tribe/Importer/Options.php

```diff
@@ -50,4 +51,34 @@ public static function getOption( $optionName, $default = '' ) {
 
 		return apply_filters( 'tribe-import-setting-' . $optionName, $value, $default );
 	}
+
+	public static function get_default_post_status( $type = 'csv' ) {
+		$options = self::getOption( 'imported_post_status', array( $type => 'publish' ) );
+
+		// Legacy for Facebook Status
+		if ( 'facebook' === $type && empty( $options['facebook'] ) ){
+			$options['facebook'] = Tribe__Events__Main::getOption( 'fb_default_status', 'publish' );
+		}
+
+		// A way to handle the legacy `imported_post_status`
+		if ( is_string( $options ) ) {
+			$options = array( $type => $options );
+		}
+
+		if ( ! isset( $options[ $type ] ) ) {
+			$options[ $type ] = apply_filters( 'tribe-import-default_post_status_non_saved', 'publish', $type );
+		}
+
+		return apply_filters( 'tribe-import-default_post_status', $options[ $type ], $type );
```

Comment:
Curious about the mix of hyphens and underscores here ... is there a reason for that?